packages/web-components/src/index.ts

```diff
@@ -3,5 +3,5 @@ export * from './counter-badge/index.js';
 export * from './progress-bar/index.js';
 export * from './spinner/index.js';
 export * from './text/index.js';
-
 export * from './theme/index.js';
+export * from './image/index.js';
```

Comment:
@halerankin lets be sure we alphabetize this - the pain will primarily be up front. This won't be as difficult when we get most components in :)